src/components/AttentionBox/AttentionBox.tsx

```diff
@@ -72,33 +75,33 @@ const AttentionBox: React.FC<AttentionBoxProps> & {
       )}
       role="alert"
     >
-      {title && (
-        <h2 className={cx(`${ATTENTION_BOX_CSS_BASE_CLASS}__title-container`, `${classNameWithType}__title-container`)}>
-          {!withoutIcon && (
-            <Icon
-              iconType={iconType}
-              ariaHidden
-              clickable={false}
-              icon={icon}
-              className={cx(
-                `${ATTENTION_BOX_CSS_BASE_CLASS}__title-container__icon`,
-                `${classNameWithType}__title-container__icon`
-              )}
-              ignoreFocusStyle
-              iconSize="24"
-              iconLabel={iconLabel}
-            />
-          )}
-          <span
+      {/* @ts-ignore */}
+      <Flex
+        /* @ts-ignore */
```

Comment:
Let me merge Flex PR https://github.com/mondaycom/monday-ui-react-core/pull/1006 first, and remove these ts-ignores?

Reply:
Merged